File: backend/components/metiers/threed/views/helpers/views_helper.js
// views_helper.js - Fonctions utilitaires avancées pour les vues Threed (JS)

function formatThreedDetails(details) {
  return `[threed] ${details}`;
}

module.exports = {
  renderTitle: (title) => `<h1>${title}</h1>`,
  renderModel: (model) => `<div>Modèle: ${model.name}</div>`,
  renderError: (msg) => `<div class='error'>${msg}</div>`,
  formatThreedDetails
};

File: backend/components/metiers/threed/views/helpers/views_helper.py
"""
views_helper.py - Fonctions utilitaires avancées pour les vues Threed (Python)
"""


def render_title(title):
    return f"<h1>{title}</h1>"


def render_model(model):
    return f"<div>Modèle: {model.get('name', None)}</div>"


def render_error(msg):
    return f"<div class='error'>{msg}</div>"
